internal/driver/onvifdiscovery.go

```diff
@@ -191,7 +191,8 @@ func executeRawProbe(conn net.Conn, params netscan.Params) ([]onvif.Device, erro
 	buf := make([]byte, bufSize)
 	// keep reading from the PacketConn until the read deadline expires or an error occurs
```

Comment:
the comment on line 192 should be changed from `keep reading from the PacketConn` to `keep reading from the connection`